src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/test/InjectDirectiveTest.cs

```diff
@@ -199,21 +199,6 @@ private DocumentIntermediateNode CreateIRDocument(RazorEngine engine, RazorCodeD
         return codeDocument.GetDocumentIntermediateNode();
     }
```

Comment:
Assuming all the deleted code is due to unused warnings.

Reply:
Yes, correct.